### ISSUE TITLE
Position's values are not allowed to exceed the Specifications in the "bit" substatement.

### ISSUE THREAD

**Issue by fredgan** [mbj4668/pyang#538]
Hi,
As described in [RFC7950 Chapter 9.7.4.2](https://tools.ietf.org/html/rfc7950#page-155)
```
   The "position" statement, which is optional, takes as an argument a
   non-negative integer value that specifies the bit's position within a
   hypothetical bit field.  The position value MUST be in the range 0 to
   4294967295, and it MUST be unique within the bits type.
```
I find that the `position` values are greater than 4294967295, but pyang(version 2.1) can't report any errors. the module is as following:
`mod1.yang`
```
module mod1 {
  yang-version 1.1;
  namespace "urn:cesnet:mod1";
  prefix abc;

  typedef my {
    type bits {
      bit one {
        description "test";
      }
      bit two {
        position 555;
      }
      bit three;
      bit four {
        position 4294967295;
      }
      bit five;
      bit six;
    }
  }
}
```
The actual result:
```
$ pyang -f yang mod1.yang
module mod1 {
  yang-version 1.1;
  namespace "urn:cesnet:mod1";
  prefix abc;

  typedef my {
    type bits {
      bit one {
        description "test";
      }
      bit two {
        position 555;
      }
      bit three;
      bit four {
        position 4294967295;
      }
      bit five;
      bit six;
    }
  }
}
```
The expected result is:
```
test/mod1.yang:18: error: the position value "4294967296" is not valid
test/mod1.yang:19: error: the position value "4294967297" is not valid
```

**Comment by fredgan** [comment]
#539

**Comment by mbj4668** [comment]
Fixed w/ #539
